src/Analyzers/Core/Analyzers/FileHeaders/AbstractFileHeaderDiagnosticAnalyzer.cs

```diff
@@ -72,9 +75,9 @@ private void HandleSyntaxTree(SyntaxTreeAnalysisContext context)
 
         private static bool CompareCopyrightText(string expectedFileHeader, string copyrightText)
         {
-            // make sure that both \n and \r\n are accepted from the settings.
-            var reformattedCopyrightTextParts = expectedFileHeader.Replace("\r\n", "\n").Split('\n');
-            var fileHeaderCopyrightTextParts = copyrightText.Replace("\r\n", "\n").Split('\n');
+            // make sure that both \n and \r\n are accepted from the settings and leading and trailing whitespace is ignored.
+            var reformattedCopyrightTextParts = expectedFileHeader.Trim().Replace("\r\n", "\n").Split('\n');
+            var fileHeaderCopyrightTextParts = copyrightText.Trim().Replace("\r\n", "\n").Split('\n');
```

Comment:
duplicated logic.  ideally extract into a `NormalizeText` helper.

Reply:
Done in 5c7972ea